coffeecard/CoffeeCard/Helpers/MobilePay/ErrorMessage/InternalServerErrorMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace CoffeeCard.Helpers.MobilePay.ErrorMessage
+{
+    public class InternalServerErrorMessage : IMobilePayErrorMessage
+    {
+        public string CorrelationId { get; set; }
+        public string Errortype { get; set; }
+        public string Message { get; set; }
+
+        public string GetErrorMessage()
+        {
+            return $"CorrelationId = {CorrelationId}, Errortype = {Errortype}, Message = {Message}";
```

Comment:
Why is this method and ToString exactly the same?

Reply:
I am not sure what was useful to get, so just duplicated for now. Any suggestions?

Reply:
I think the output is fine, maybe just call the toString method in getError message to avoid duplication?